packages/bitcore-wallet-service/src/lib/blockchainexplorers/v8.ts

```diff
@@ -122,7 +120,7 @@ export class V8 {
       name: wallet.id,
       pubKey: wallet.beAuthPublicKey2,
       network: this.v8network,
-      chain: this.coin
+      chain: this.coin // lowerCase?
```

Comment:
I think this should be this.chain

Reply:
I agree. but it seems we are sending it lowercase now, so I didnt wanted to modify it.

Is it possible we have something on bitcore-node what allows chain to be lowerCased on wallet registration?

Reply:
```
  let { chain, network } = req.params;
  let { name, pubKey, path, singleAddress } = req.body;
```

Looks like it's coming from the URL, and not the request body

Reply:
ic, ok, ill change it anyways so it is clear.